modules/ROOT/pages/database-administration/composite-databases/querying-composite-databases.adoc

```diff
@@ -17,7 +15,40 @@ Queries submitted to a Composite database may contain several `USE` clauses that
 
 Each constituent graph is named after the alias that introduces it into the Composite database.
 
-The examples assume the same setup as the one created in xref:composite-databases/administration.adoc[].
+[[composite-databases-queries-graph-set-up]]
+== Graph set-up
```

Comment:
![image](https://github.com/neo4j/docs-operations/assets/7591575/a500ed62-e29d-4f56-994d-48809b64ef68)
this feels like an odd order/hierarchy to me. It would feel more natural to me to have `Graph set-up` be the first one and then have `Graph selection` which would have all the subsections currently living under `Graph set-up`